Formula/d/dpcmd.rb

```diff
@@ -27,6 +27,7 @@ class Dpcmd < Formula
   def install
     system "make"
     bin.install "dpcmd"
+    include.install "ChipInfoDb.dedicfg"
```

Comment:
Can we add a test that fails when this program is missing?

Reply:
The test does fail, but only if the programmer (a usb device) was attached unfortunately. I haven't found a way to make it show an error without it attached.

Reply:
Oh sorry, actually, the current test won't fail even if the device is connected.

```
> dpcmd -rSTDOUT -a0x100 -l0x23

DpCmd Linux 1.14.20.01 Engine Version:
Last Built on Jan 11 2024





> dpcmd --list

DpCmd Linux 1.14.20.01 Engine Version:
Last Built on Jan 11 2024


Error: chip not identified or chip is not supported.


```

Reply:
The exit code for the first one was `1` and the second one was `10`.